template/src/components/header/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { h } from "preact";
+import preact, { h } from "preact";
```

Comment:
what's that about?

Reply:
Just the IDE fixing a poor typing that I made. Will fix.